app/src/main/java/live/hms/app2/ui/meeting/SettingsBottomSheet.kt

```diff
@@ -271,6 +271,9 @@ class SettingsBottomSheet(
             binding.btnShowStats.visibility = View.GONE
             binding.btnBrb.visibility = View.GONE
         }
+        if (meetingViewModel.hmsSDK.getLocalPeer()?.isWebrtcPeer() == true || meetingViewModel.isHlsRunning()){
```

Comment:
Can we keep the visibility on always for `btnShowStats` since both modes have stas for nerds

Reply:
actually this one i wanted to discuss , that is there some scenario that we expect where stats for nerds isnt visible ?

Reply:
Stats for nerds will be visible always.. for webrtc as well as non webrtc users

Reply:
resolved